react/features/analytics/actions.js

```diff
@@ -0,0 +1,119 @@
+import JitsiMeetJS from '../base/lib-jitsi-meet';
+import { loadScript } from '../base/util';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * Loads the analytics scripts and inits JitsiMeetJS.analytics by setting
+ * permanent properties and setting the handlers from the loaded scripts.
+ * NOTE: Has to be used after JitsiMeetJS.init. Otherwise analytics will be
+ * null.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @returns {void}
+ */
+export function initAnalytics({ getState }) {
+    window.analyticsHandlers = [];
```

Comment:
Is there really no other way? Can't we store them in redux?

Reply:
I didn't get that, what do you mean?